packages/modules/workflow-engine-redis/src/utils/workflow-orchestrator-storage.ts

```diff
@@ -76,6 +94,11 @@ export class RedisDistributedTransactionStorage
       JobType.TRANSACTION_TIMEOUT,
     ]
 
+    const workerOptions = {
+      connection:
+        this.redisWorkerConnection /*, runRetryDelay: 100000 for tests */,
+    }
+
     this.worker = new Worker(
```

Comment:
Isn't it better for this worker/queue only run on a server that is in `shared` or `worker` mode? Like that, the server only deals with HTTP, and the worker deals with async workflows, jobs, and subscribers

Reply:
I think it is fine for workflow to be executed on the server instance as well no? I don't think you should be forced to have a second instance of medusa to run async workflows wdyt?

Reply:
that would be a breaking change in the users infrastructure/api whereas it is not for jobs and subscribers since they are not loaded in server instances. I do agree with you overall as it would prevent increasing the pressure on the event loop more that increasing performances but I understand your point 👍 

@olivermrbl since this suggestion would have breaking implication on users deployment wdyt?